webpack.config.js

```diff
@@ -71,10 +71,7 @@ module.exports = (params = {}) => {
               }
             },
             {
-              loader: 'resolve-url-loader',
-              options: {
-                keepQuery: true
-              }
+              loader: 'resolve-url-loader'
```

Comment:
The keepQuery behaviour is now the default, the keepQuery option has been removed.